### ISSUE TITLE
detail-page for issue

### ISSUE THREAD

**Issue by Cahllagerfeld** [EddieHubCommunity/good-first-issue-finder#36]

**Comment by bhaskar2840** [comment]
- yes! surely it would be a great step.

**Comment by Cahllagerfeld** [comment]
The Issues I'm seeing with this, is that I'm not sure if its worth to rebuild kind of the thing Github already provides. It kinda feels we're reinventing the wheel which isnt necessary

**Comment by krshkun** [comment]
I agree with Cahl, the most I think we can do is have first a popup that gives the user some information about the issue instead of directly opening it in github

**Comment by Cahllagerfeld** [comment]
I'm on the fence about this issue and might close it. I'm actually not keen on rebuilding features that Github already provides. I see no reason to reinvent the wheel here
